data/mods/No_Hope/items.json

```diff
@@ -266,5 +266,26 @@
     "name": { "str": "broken laser turret" },
     "weight": "110 kg",
     "copy-from": "broken_turret"
+  },
+  {
+    "id": "lobotomizer",
```

Comment:
You can ditch this entry (no hope will just use it because it's in obsolete.json).

Reply:
done.